### PR TITLE
[Release 4.5] Bug 1880315: Fix bug in reflector not recovering from "Too large resource version"

### PR DESCRIPTION
**Pull request by sgreene570**
This bump imports the recent fix in reflector/informer that was made in upstream client-go

kubernetes/kubernetes#92688

As per https://bugzilla.redhat.com/show_bug.cgi?id=1880315.

Note this was already fixed in 4.6 with the bump to 1.19. https://github.com/openshift/cluster-ingress-operator/pull/453

/assign @miciah @danehans 
/cc @frobware @knobunc @miheer

### CODE REVIEW

**Comment by sgreene570** [comment]
`make verify` works locally for me with `go1.13.12`
Let me try again...
/test verify

**Comment by sgreene570** [comment]
https://github.com/openshift/cluster-ingress-operator/pull/462

Bumping `openshift/library-go` to resolve 
`bitbucket.org/ww/goautoneg@v0.0.0-20120707110453-75cd24fc2f2c: reading https://api.bitbucket.org/2.0/repositories/ww/goautoneg?fields=scm: 404 Not Found`

**Comment by Miciah** [comment]
> This bump imports the recent fix in reflector/informer that was made in upstream client-go
> 
> [kubernetes/kubernetes#92688](https://github.com/kubernetes/kubernetes/pull/92688)

Also https://github.com/kubernetes/kubernetes/pull/94430.

Could you put these references in the commit message?  (For the commit message, I'd link to the commits to avoid GitHub's overzealous cross-referencing: https://github.com/kubernetes/kubernetes/pull/92688/commits/8012722d626d50fd18059ec5af6d195a7dc180c1 and https://github.com/kubernetes/kubernetes/pull/94430/commits/744dd65b388e9fc17f53072db35c38748a83e777.)

**Comment by Miciah** [comment]
Please explain the library-go bump in the commit message too for the benefit of people looking through Git history.

**Comment by sgreene570** [comment]
Thanks for the reminder @Miciah . I've updated the commit messages. Ill do the same for the dns operator PR.

**Comment by Miciah** [comment]
Thanks!
/lgtm
